migtests/tests/import-file/run-import-file-test

```diff
@@ -223,9 +223,9 @@ main() {
 			--file-table-map "csv_with_header_test.csv:gcs_csv_with_header" --has-header true
 
 	# Test larger import from GCS buckets
-	step "Import large data file from GCS: volume.csv-> gcs_volume"
```

Comment:
volume.csv had header which as comma-delimited, and data which was tab-delimited. 
With the change in this PR, the columns were looked up in the attribute name registry, and was therefore failing. 

I fixed the file, and instead of overwriting, I created a volume1.csv just to be on the safe side!